src/Microsoft.Health.Dicom.Core/Features/Query/QueryResource.cs

```diff
@@ -7,11 +7,11 @@ namespace Microsoft.Health.Dicom.Core.Features.Query
 {
     public enum QueryResource
```

Comment:
nit: as we discussed, I think it's ok to leave the numbers out (as they are the same values if left unspecified), but let's leave a comment about how the order works